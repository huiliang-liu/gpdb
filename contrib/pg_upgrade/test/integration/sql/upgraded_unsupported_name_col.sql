SELECT * FROM table_with_name_column;
SELECT attname, atttypid::regtype FROM pg_attribute WHERE attrelid='table_with_name_column'::regclass and attname='a_name';
