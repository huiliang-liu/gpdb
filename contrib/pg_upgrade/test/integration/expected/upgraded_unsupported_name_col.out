SELECT * FROM table_with_name_column;
 a | a_name  
---+---------
 1 | abc def 
 2 | ghi jkl 
(2 rows)
SELECT attname, atttypid::regtype FROM pg_attribute WHERE attrelid='table_with_name_column'::regclass and attname='a_name';
 attname | atttypid          
---------+-------------------
 a_name  | character varying 
(1 row)
